.gitignore

```diff
@@ -46,11 +45,14 @@ temp-testng-customsuite.xml
 .settings
 .springBeans
 /components/tools/OmeroWeb/omeroweb/media/webtest
+/components/tools/OmeroWeb/omeroweb/static/
```

Comment:
I could see moving these down to OmeroWeb/.gitignore (or one deeper). Especially when/if OmeroWeb is extracted.